components/Callout/index.tsx

```diff
@@ -31,9 +31,9 @@ const Callout = (props: Props) => {
     <blockquote {...attributes} className={`callout callout_${theme}`} theme={icon}>
       <h3 className={`callout-heading${empty ? ' empty' : ''}`}>
         <span className="callout-icon">{icon}</span>
-        {empty || children[0]}
+        {empty || React.Children.toArray(children)[0]}
       </h3>
-      {empty ? children : React.Children.toArray(children).slice(1)}
+      {React.Children.toArray(children).slice(1)}
```

Comment:
Why do we not need this ternary logic anymore?

Reply:
It was a little simpler to leave the title/heading as an empty child. But we could just as easier refactor the code not to.